engine/src/main/java/org/terasology/engine/modes/StateLoading.java

```diff
@@ -142,13 +139,10 @@ public void init(GameEngine engine) {
 
     private void initClient() {
         loadProcesses.add(new JoinServer(context, gameManifest, joinStatus));
-        loadProcesses.add(new CacheTextures());
```

Comment:
Are these processes/steps obsolete, or _just_ not used (yet)? I would think they are/were there for a reason, but unfortunately I don't know that reason… 🙄

Reply:
Since at least 2013, they don't do anything at all it seems to me. The code in the classes was always just moved/refitted to new interface without any change to functionality, which might well predate any piece of current framework. To figure out what those were actually meant to do, I would have to get myself a workspace from back then/find the PR with this code and hope for explaining notes

Reply:
Let's make sure that we find this PR again in case we later figure out that something breaks. Code that's doing nothing since 2013 might have some right to continue do nothing (😅) but I think we should get rid of it now 👍